.github/workflows/k8s-testing.yml

```diff
@@ -108,9 +108,22 @@ jobs:
 
     strategy:
       matrix:
-        databases: [pgsql, mysql]
-        brokers: [redis, rabbit]
-        k8s: ['v1.16.0','v1.20.0']
+        include:
```

Comment:
Personally I'm fun of  deep testing approach, to test all combination.

Reply:
I don't think we really need it and with "Green IT" in mind it better to use less resources.

Reply:
Ok, I can survive it :)